packages/flutter/lib/src/foundation/basic_types.dart

```diff
@@ -7,8 +7,8 @@ import 'dart:collection';
 
 // COMMON SIGNATURES
 
-export 'dart:ui' show VoidCallback;
-export 'bitfield.dart' if (dart.library.html) 'bitfield_unsupported.dart';
+export 'ui_stub.dart'
+  if (dart.library.ui) 'dart:ui' show VoidCallback;
```

Comment:
it's weird that we only limit it to `show VoidCallback` on the second line... is that conditional import syntax, or should we have the `show` on both lines?

Reply:
This is the correct syntax, the VoidCallback is shown from both libraries.